qa/automation/Starynina.Scenarios/3. Send CV/3.2.Send CV.feature

```diff
@@ -0,0 +1,136 @@
+	| one word (nina)                                        | "Invalid name format"    |
+	| one word, space, numberes (nina 1988)                  | "Invalid name format"    |
+	| special symbols, space, special symbols (/*-+?:%; 909) | "Invalid name format"    |
+	| two characters with space in between (n s)             | accepted                 | 
+	| two words in English (nina star)                       | accepted                 | 
+	| two words in Cyrillic                                  | accepted                 | 
+
+@negative
+Scenario: 3.2_06 User enters name with more than maximum characters in the CV form 
+	When user enters name of 71 characters or more in the "Name" field
+	Then the error messages "The maximum number of characters is 70" for "Name" field is appeared
+	And the "Name" field should be highligted in red
+
+Scenario Outline: 3.2_07 User enters various types of email in the CV form
+	When user enters <email> in the "Email" field
+	Then the result <result> should appear
+
+	Examples:
+	| email                                | result                   |
+	| nothing entered ()                   | "This field is required" |
+	| email without @ (nina.gmail.com)     | "Invalid name format"    |
+	| numbers (123)                        | "Invalid name format"    |
+	| email without two @ (nina@mail@.com) | "Invalid name format"    |
+	| email with .com (star@gmail.com)     | accepted                 | 
+	| email with .ru (star@mail.ru)        | accepted                 | 
+	| email with wordInCyrillic@gmail.com  | accepted                 | 
+
+@negative
+Scenario: 3.2_08 User enters email with more than maximum characters in the CV form 
+	When user enters name of 71 characters or more in the "Email" field
+	Then the error messages "The maximum number of characters is 70" for "Name" field is appeared
+	And the "Email" field should be highligted in red
+
+@negative
+Scenario: 3.2_09 User enters message with more than maximum characters in the CV form 
+	When user enters data of 501 characters or more in the "Message" field
+	Then the error messages "The maximum number of characters is 500" for "Message" field should appear
+	And the "Message" field should be highligted in red
+
+Scenario Outline: 3.2_10 User can't attach CV file of invalid file format
+	Given user clicks the "Attach CV" button
+	When user chooses file of <invalid format>
+	Then the error messages "Something went wrong Invalid file format. Please try PDF, DOC, DOCS or RTF" should appear
+
+	Examples:
+	| invalid format |
+	| JPEG           |
+	| TXT            |
+	| AVI            |
+
+Scenario: 3.2_11 User can't attach CV file of wrong file size
+	Given user clicks the "Attach CV" button
+	When user chooses file of valid DOC format 10.1 Mb size
+	Then the error messages "Something went wrong Max file size 10 Mb" is appeared
+
+Scenario: 3.2_12 User can fill the CV form with valid text data
```

Comment:
No need for such a scenario. You should fill and send.

Reply:
Okay, deleted